templates/module/src/entity-content-route-provider.php.twig

```diff
@@ -161,7 +161,7 @@ class {{ entity_class }}HtmlRouteProvider extends AdminHtmlRouteProvider {% endb
       $route = new Route("/admin/structure/{$entity_type->id()}/settings");
       $route
         ->setDefaults([
-          '_form' => 'Drupal\{{ module }}\Controller\{{ entity_class }}SettingsForm',
+          '_form' => 'Drupal\{{ module }}\Form\{{ entity_class }}SettingsForm',
```

Comment:
I run `generate:entity:content` and here is the list of generated files:
Generated or updated files
 Site path: /Users/shawn/Sites/entity-demo/
 1 - modules/custom/entity_test/entity_test.permissions.yml
 2 - modules/custom/entity_test/entity_test.links.menu.yml
 3 - modules/custom/entity_test/entity_test.links.task.yml
 4 - modules/custom/entity_test/entity_test.links.action.yml
 5 - modules/custom/entity_test/src/DefaultEntityInterface.php
 6 - modules/custom/entity_test/src/DefaultEntityAccessControlHandler.php
 7 - modules/custom/entity_test/src/Entity/DefaultEntity.php
 8 - modules/custom/entity_test/src/DefaultEntityHtmlRouteProvider.php
 9 - modules/custom/entity_test/src/Entity/DefaultEntityViewsData.php
 10 - modules/custom/entity_test/src/DefaultEntityListBuilder.php
 11 - modules/custom/entity_test/src/Form/DefaultEntitySettingsForm.php
 12 - modules/custom/entity_test/src/Form/DefaultEntityForm.php
 13 - modules/custom/entity_test/src/Form/DefaultEntityDeleteForm.php
 14 - modules/custom/entity_test/default_entity.page.inc
 15 - modules/custom/entity_test/templates/default_entity.html.twig

Using version 0.10.6 - I don't see entity_test/Controller/...  And if I ask phpStorm to show me all classes with DefaultEntity I get:

![defaultentity-class-list](https://cloud.githubusercontent.com/assets/414635/12697754/e9840bb6-c758-11e5-93d7-8c1f0935d8ea.png)

And if I open DefaultEntityHtmlRouteProvider.php and copy the class name there, `Drupal\entity_test\Controller\DefaultEntityAddController` and look for it I get no match.

Am I not looking at this correctly?